components/Button.js

```diff
@@ -1,5 +1,5 @@
 function Button(props) {
-    console.log("hello")
+    console.log('hello')
```

Comment:
changed it according to victor's feedback

Reply:
good